packages/docs/overview.md

```diff
@@ -10,7 +10,7 @@ This page provides some background on blockchain technology and explores the Cel
 
 A **blockchain** or **cryptographic network** is a broad term used to describe a database maintained by a distributed set of computers that do not share a trust relationship or common ownership. This arrangement is referred to as **decentralized**. The content of a blockchain's database, or **ledger**, is authenticated using cryptographic techniques, preventing its contents being added to, edited or removed except according to a protocol operated by the network as a whole.
 
-The code of the Celo Blockchain has shared ancestry with [Ethereum](https://www.ethereum.org), blockchain software for building general-purpose decentralized applications. Celo differs from Ethereum in several important areas as described in the following sections. However it inherits a number of key concepts.
+The code of the Celo Blockchain has shared ancestry with [Ethereum](https://www.ethereum.org), a blockchain software for building general-purpose decentralized applications. Celo differs from Ethereum in several important areas as described in the following sections. However, it inherits a number of key concepts.
```

Comment:
Isn't "blockchain software" correct since it's uncountable?

Reply:
Makes sense to me